xml/obs_ag_source_publish.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.1"
+ xml:id="_source_publish">
+ <title>Source Publisher</title>
+ <para>The job of the source publish service is to publish all sources for 
+directly before published binaries. This will also the sources of repackaged
+binaries. For example the sources of rpms which are used inside of product, 
+appliance or container images. A prerequisite for this is that OBS and the
+build tooling supports content tracking.</para>
+
+ <sect2 xml:id="_source_publish_configuration">
+  <title>Configuring Source Publisher</title>
+  <para>The source publishing can be configured via the file
+    <emphasis>/usr/lib/obs/server/BSConfig.pm</emphasis>, where it can be 
+    global enabled or just for some projects. It is possible to use regular
+   expressions here.</para>
+  <para>The publishing can be enabled by defining the rsync module to push the content:</para>
+  <screen language="perl">our $sourcepublish_sync = 'rsync://your_rsync_server/rsync_module'; </screen>
+  <para>By default every project get published, but it is possible to define a whitelist via:</para>
+  <screen language="perl">our $sourcepublish_filter = [ "openSUSE:.*", "SUSE:.*" ]; </screen>
+ </sect2>
+
+ <sect2 xml:id="_source_publish_considerations">
+  <title>Considerations</title>
+  <para>The source publishing is publishing the sources as they are hosted in &obs;. This
+        means these are the unprocessed sources and the content is not identical to the
+        content of source rpms for example. Instead these are the sources which are the
+        base for source rpms.</para>
+  <para>As a consequence hints like NoSource: tags in spec files are ignored. The only
+        way to disable publishing for the user is to disable access or sourceaccess via
+        the flags.</para>
+  <para>The filesystem structure is <emphasis>$project/$package/$srcmd5/</emphasis>. A &disturl;
+        inside of binary builds can be used to find the right sources.</para>
+  <para>&obs; will care for de-duplication on the rsync server. This must get implemented there.</para>
```

Comment:
* wording correction: `&obs; will take care of deduplication ...`
* I am a bit confused by the final sentence -- is this feature already implemented? If so, I'd just remove that sentence.

Reply:
the rsync server is not part of OBS. this is something admin has to take care of.